src/mono/mono/mini/aot-compiler.c

```diff
@@ -7005,6 +7005,13 @@ encode_patch (MonoAotCompile *acfg, MonoJumpInfo *patch_info, guint8 *buf, guint
 		encode_value (((guint32 *)patch_info->data.target) [MINI_LS_WORD_IDX], p, &p);
 		encode_value (((guint32 *)patch_info->data.target) [MINI_MS_WORD_IDX], p, &p);
 		break;
+	case MONO_PATCH_INFO_X128:
+	case MONO_PATCH_INFO_X128_GOT:
+		encode_value (((guint32 *)patch_info->data.target) [(MINI_LS_WORD_IDX * 2) + MINI_LS_WORD_IDX], p, &p);
+		encode_value (((guint32 *)patch_info->data.target) [(MINI_LS_WORD_IDX * 2) + MINI_MS_WORD_IDX], p, &p);
+		encode_value (((guint32 *)patch_info->data.target) [(MINI_MS_WORD_IDX * 2) + MINI_LS_WORD_IDX], p, &p);
```

Comment:
The indexes are the same.

Reply:
Could you clarify? It's doing:
```
LS * 2 + LS
LS * 2 + MS
MS * 2 + LS
MS * 2 + MS
```

On a little endian system that's:
```
0 * 2 + 0 = 0
0 * 2 + 1 = 1
1 * 2 + 0 = 2
1 * 2 + 1 = 3
```

Reply:
Ah, nevermind it's `p` that is taking the wrong index

Reply:
Misread the changes.

Reply:
Actually, no. Not seeing the issue. `MONO_PATH_INFO_R8` also doesn't increment `p` it does:
```
		encode_value (((guint32 *)patch_info->data.target) [MINI_LS_WORD_IDX], p, &p);
		encode_value (((guint32 *)patch_info->data.target) [MINI_MS_WORD_IDX], p, &p);
```